src/store/todo/todo.slice.ts

```diff
@@ -3,7 +3,7 @@ import { Todo, TodoState } from 'types/todo';
 import { fetchTodos, deleteTodo, addNewTodo, changeTodoStatus } from './todo.actions';
 
 const initialState: TodoState = {
-  todos: [],
+  todos: null,
```

Comment:
why do you use one more type for todos?

Reply:
because I need fetch data only when we don't have todos at all